### ISSUE TITLE
1.18.2 - server incompatibility

### ISSUE THREAD

**Issue by antirain1** [ThinkingStudios/MaLiLib-Forge#7]
when starting the server, it gives a similar error:
MaFgLib (malilib) has failed to load correctly
§7java.lang.RuntimeException: Attempted to load class net/minecraft/client/gui/screens/Screen for invalid dist DEDICATED_SERVER,
Forgematica (litematica) has failed to load correctly
![chrome_MOrKT5Lau5](https://github.com/ThinkingStudios/MaLiLib-Forge/assets/135417333/0b5dfb30-c80f-4563-abed-6cd2fbc140e9)

**Comment by TexBlock** [comment]
This mod is only client side

**Comment by antirain1** [comment]
> This mod is only client side

yes, but for some reason, when you try to connect to the server, it writes that the mod sheet does not match (if you add litematica and malilib to your port)

**Comment by antirain1** [comment]
> > This mod is only client side
> 
> yes, but for some reason, when you try to connect to the server, it writes that the mod sheet does not match (if you add litematica and malilib to your port)

![chrome_HbhGXgP3lh](https://github.com/ThinkingStudios/MaLiLib-Forge/assets/135417333/3f0d1ed8-9b65-471b-8416-07f18c7f7c07)
this happens only when I just add MaFgLib-0.1.4fix-mc1.18.2 and forgematica-0.1.4-mc1.18.2

**Comment by TexBlock** [comment]
I will try to fix it in the next few days.

**Comment by antirain1** [comment]
> I will try to fix it in the next few days.

thank you!
谢谢!

**Comment by TexBlock** [comment]
https://github.com/ThinkingStudios/MaLiLib-Forge/releases/tag/v0.1.4fix2-mc1.18.2

**Comment by antirain1** [comment]
> https://github.com/ThinkingStudios/MaLiLib-Forge/releases/tag/v0.1.4fix2-mc1.18.2

thank you very much, but what about forgematica itself, not the MaLiLib library? 
I downloaded the last version (forgematica-0.1.4-mc1.18.2) with a new library (MaFgLib-0.1.4fix2-mc1.18.2.jar ) 
I still get the mods incompatibility error, which indicates that the mod is still not client-side :(

**Comment by TexBlock** [comment]
can you give me log?

**Comment by TexBlock** [comment]
They're not installed on the server, are they?

**Comment by antirain1** [comment]
> They're not installed on the server, are they?

Yes, **MaFgLib-0.1.4fix2-mc1.18.2.jar** and **forgematica-0.1.4-mc1.18.2.jar** they were not installed on the server. 
These mods were installed on the client, and when entering the server list, the server displays that an incompatible package of mods, i.e. +2, it considers these two mods mandatory for installation on the server.

Then I tried to install the following mods on the server:
- MaFgLib-0.1.4fix2-mc1.18.2.jar (updated)
- forgematica-0.1.4-mc1.18.2.jar
Bottom line = crashes the server, 
I'll attach the logs now. 
https://mclo.gs/d4TXn5v - debug.log
https://mclo.gs/xnqic6N - latest.log

_Maybe the reason is in the un-updated version **forgematica-0.1.4-mc1.18.2.jar** which has not been fixed?_

**Comment by TexBlock** [comment]
fixed in latest version

**Comment by antirain1** [comment]
> fixed in latest version

i love you man
